### PR TITLE
ngfw-13410 portMatcher VType modified for ip port validation bug

### PR DESCRIPTION
**Pull request by DhirajM09**
Currently in NGFW router rules like NAT, Bypass and filter rules IP address port numbers 0 and 65536 are accepted.
Modified validation function to limit the range to **1-65536**.

While doing analysis found validation passing for `<1` and `>65535` inputs though these are invalid inputs. hence changed portMatcher function to accept only valid inputs like `<=1`(only one value is valid i.e. 1) and '>=65535' (only one value is valid i.e. 65535).

Local Testing Screenshots

Screenshot from 2024-02-08 18-52-23

Screenshot from 2024-02-08 18-50-37

Screenshot from 2024-02-08 18-56-11

Screenshot from 2024-02-08 18-56-27

Screenshot from 2024-02-08 18-57-29

Screenshot from 2024-02-08 18-57-13

### CODE REVIEW

File: uvm/servlets/admin/app/overrides/form/field/VTypes.js [hunk 1]
```diff
@@ -243,11 +243,11 @@ Ext.define('Ung.overrides.form.field.VTypes', {
         case 'any':
             return true;
         default:
-            if (val.indexOf('>') !== -1 && val.indexOf(',') === -1) {
-                return this.isSinglePortValid( val.substring( val.indexOf('>') + 1 ));
+            if (val.indexOf('>=') !== -1 && val.indexOf(',') === -1) {
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/app/overrides/form/field/VTypes.js:246]: The val check for >=, >. <, and <= are all values the user is able to specify so we need to support those as well.  I think this code could break that, so be sure to retain support for the "non =" operations as well.

File: uvm/servlets/admin/app/overrides/form/field/VTypes.js [hunk 2]
```diff
@@ -243,11 +243,11 @@ Ext.define('Ung.overrides.form.field.VTypes', {
...
+                return this.isSinglePortValid( val.substring( val.indexOf('>=') + 2 ));
             }
-            if (val.indexOf('<') !== -1 && val.indexOf(',') === -1) {
-                return this.isSinglePortValid( val.substring( val.indexOf('<') + 1 ));
+            if (val.indexOf('<=') !== -1 && val.indexOf(',') === -1) {
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/app/overrides/form/field/VTypes.js:249]: The val check for >=, >. <, and <= are all values the user is able to specify so we need to support those as well.  I think this code could break that, so be sure to retain support for the "non =" operations as well.

File: uvm/servlets/admin/app/overrides/form/field/VTypes.js (hunk 1 above)
Comment (DhirajM09) [reply uvm/servlets/admin/app/overrides/form/field/VTypes.js:246]: Added code to handle `>` and `<` Symbols

Screenshot from 2024-02-08 20-42-01

Screenshot from 2024-02-08 20-41-42

Screenshot from 2024-02-08 20-41-06

File: uvm/servlets/admin/app/overrides/form/field/VTypes.js (hunk 2 above)
Comment (DhirajM09) [reply uvm/servlets/admin/app/overrides/form/field/VTypes.js:249]: Added code to handle `>` and `<` Symbols

Screenshot from 2024-02-08 20-42-57

Screenshot from 2024-02-08 20-42-42

Screenshot from 2024-02-08 20-42-26

**Review by cblaise-untangle — APPROVED** [review]

**Pull request merged**